packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -271,23 +271,25 @@ export const _FileDownloadCards = (props: _FileDownloadCards): JSX.Element => {
               return true;
             })
             .map((file) => (
-              <_FileCard
-                fileName={file.name}
-                key={file.name}
-                fileExtension={file.extension}
-                actionIcon={
-                  showSpinner ? (
-                    <Spinner size={SpinnerSize.medium} aria-live={'polite'} role={'status'} />
-                  ) : true &&
-                    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
-                    isShowDownloadIcon(file) ? (
-                    <IconButton className={iconButtonClassName} ariaLabel={downloadFileButtonString()}>
-                      <DownloadIconTrampoline />
-                    </IconButton>
-                  ) : undefined
-                }
-                actionHandler={() => fileDownloadHandler(userId, file)}
-              />
+              <TooltipHost content={downloadFileButtonString()} key={file.name}>
```

Comment:
Is file.name as the key sufficient enough?
Can Contoso upload a file with the same name?

Reply:
I don't think there's any check on this for when upload, but this is existing logic with _FileCard, just extracted out an layer to work with TooltipHost. Not in scope for this tooltip bugfix